src/missing_deps.h

```diff
@@ -60,8 +60,6 @@ namespace std {
 template <>
 struct hash<EdgePair> {
   size_t operator()(const EdgePair& k) const {
-    // Compute individual hash values for first, second and third
-    // http://stackoverflow.com/a/1646913/126995
     uintptr_t uint_from = (uintptr_t)(k.from);
     uintptr_t uint_to = (uintptr_t)(k.to);
     return hash<uintptr_t>()(uint_from ^ (uint_to >> 3));
```

Comment:
this is just a simplistic "hash" to get an unordered_map going, ended up plenty faster than map